### PR TITLE
DCM-70: Use loggers instead of println commands in dhis connector service

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
### The issue I worked on

See: https://issues.openmrs.org/browse/DCM-70

### Description of what I changed:
* Replace all `println` commands with loggers.

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java [hunk 1]
```diff
@@ -819,7 +819,7 @@ public DHISDataSet getDHISDataSetById(String id) {
 			dataSet = mapper.readValue(node.toString(), DHISDataSet.class);
 		}
 		catch (Exception ex) {
-			System.out.print(ex.getMessage());
+			log.error(ex.getMessage());
```
Comment (jayasanka-sack) [inline api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java:822]: Just log the exception here

File: api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java [hunk 2]
```diff
@@ -1074,15 +1074,15 @@ public String getDHIS2APIBackupPath() {
...
 			out = new ZipOutputStream(new FileOutputStream(zipFile));
 			
-			System.out.println("Creating : " + zipFile);
+			log.info("Creating : " + zipFile);
 			addDHIS2APIDirectories(dirObj, out, sourceDirectory);
 			out.close();
 		}
 		catch (FileNotFoundException e) {
-			e.printStackTrace();
+			log.error("Error ",e);
```
Comment (jayasanka-sack) [inline api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java:1082]:
```suggestion
			log.error("Error", e);
```

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Good job @kumuditha-udayanga ! 

Added some minor comments

File: api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java (hunk 1 above)
Comment (kumuditha-udayanga) [reply api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java]: Sure

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done, @kumuditha-udayanga ! 🤗

**Pull request merged**